main.cpp

```diff
@@ -1,2 +1,5 @@
 int main() {
+  int a;
```

Comment:
warning: variable 'a' is not initialized [cppcoreguidelines-init-variables]

```suggestion
  int a = 0;
```